Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -47,6 +47,18 @@ public function nearestContentCollection(NodeInterface $node, $nodePath)
         }
     }
 
+    /**
+     * Generate a label for a node with a chaining mechanism. To be used in nodetype definitions.
+     *
+     * @param string|null $label
+     * @param NodeInterface|null $node
+     * @return string
+     */
+    public function labelForNode(string $label = null, NodeInterface $node = null): NodeLabelToken
```

Comment:
The method signature is a bit wird to me. Beeing in the node helper i would expect the Node to be the main (non optional) argument. 

Maybe with a second parameter `array $properties = ['title']` that provides a list of properties that are checked one after another.